client/js/components/statement/listStatements/ListStatements.vue

```diff
@@ -778,8 +778,8 @@ export default {
      * Returns the hash of the original statement attachment
      */
     getOriginalPdfAttachmentHash (el) {
-      if (el.hasRelationship('attachments')) {
-        const originalAttachment = Object.values(el.relationships.attachments.list())
+      if (el.hasRelationship('genericAttachments')) {
+        const originalAttachment = Object.values(el.relationships.genericAttachments.list())
```

Comment:
wait. shouldn't this be `sourceAttachments` ??

Reply:
hm, does not look like that --> ![image](https://github.com/user-attachments/assets/397dfcf3-a1f4-4865-a2d7-345da16f7217)